frontend_vue/src/components/base/_BaseSwitch.vue

```diff
@@ -0,0 +1,107 @@
+<!-- eslint-disable vuejs-accessibility/label-has-for -->
+<template>
+  <div class="flex flex-col-reverse justify-between">
+    <input
+      v-bind="$attrs"
+      :id="id"
+      v-model="model"
+      type="checkbox"
+      role="switch"
+      class="toggle"
+      :checked="model"
+      :aria-checked="model"
+    />
+    <label :for="id">{{ label }}</label>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  id: string;
+  label: string;
+}>();
+
+const model = defineModel<boolean>();
+</script>
+
+<style scoped>
+input[type='checkbox'].toggle {
+  opacity: 0;
+  position: absolute;
+  left: -9000px;
+  top: -9000px;
```

Comment:
O_O

Reply:
it's a trick to remove it from the view 😶‍🌫️